conans/client/tools/oss.py

```diff
@@ -275,9 +277,9 @@ def get_freebsd_version():
     def get_solaris_version_name(version):
         if not version:
             return None
-        elif version.minor() == "5.10":
+        elif version.minor == "10":
```

Comment:
Shouldn't we keep the major just in case the "6" comes?

Reply:
Do you really think that a Solaris major version is going to be released? 😄 

And if it is released, still a fix would be necessary to add the new model anyway. But sure, no problem, I can add a check on the major too.

Reply:
Fixed, added 5 major check